keyboards/cannonkeys/practice65/config.h

```diff
@@ -54,6 +54,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define RGB_DI_PIN B15
 #define RGBLED_NUM 20
 #define WS2812_SPI SPID2
+#define WS2812_SPI_SCK_PIN B13
```

Comment:
Again, `B13` is in `MATRIX_COL_PINS` on this board, so this change is definitely not correct for this hardware.

Reply:
you can leave this one as is - we don't sell Practice65s anymore (only V2).

this is f103 powered, and worked without SCK set I think.